src/themes/themes.wac.css

```diff
@@ -0,0 +1,256 @@
+﻿.red {
```

Comment:
Please put some comment why and how you created this style and the difference from the original style.

Reply:
Not sure about this. It's a copy of theme.blue.json but that's not a relevant information from this point onwards and the difference will only grow.